src/backend/distributed/transaction/transaction_management.c

```diff
@@ -102,6 +104,12 @@ MemoryContext CommitContext = NULL;
  */
 bool ShouldCoordinatedTransactionUse2PC = false;
 
+/*
+ * Distribution function arguments when delgated using forcePushdown flag.
+ */
+Const *AllowedDistributionColumnValue = NULL;
+bool ShouldAllowRestricted2PC = false;
```

Comment:
should have a comment describing what it does

Reply:
Done.